incubator/jaeger/templates/spark-cronjob.yaml

```diff
@@ -17,6 +17,8 @@ metadata:
 {{- end }}
 spec:
   schedule: {{ .Values.spark.schedule | quote }}
+  successfulJobsHistoryLimit: {{ default 5 .Values.spark.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ default 5 .Values.spark.failedJobsHistoryLimit }}
```

Comment:
Please add defaults to `values.yaml` instead of using the `default` function.

Reply:
👍